### PR TITLE
Find Bigfoot community workshop

### PR DESCRIPTION
**Pull request by ad510**
As promised, here's a workshop about making a game about finding Bigfoot.

This might be pretty unpolished because I wrote a significant amount of it in a few hours at PennApps. Do you think the workshop focuses on the right kinds of things?

It might also be on the short side. The original plan was to animate Bigfoot like at http://andrewd.50webs.com/bigfoot/1IllustriousBigfoot/1findBigfoot.htm but I didn't include that in the workshop for the sake of time. I may add that back in the future.

Also, where do you get all the celebratory images? Currently I'm reusing the ones from the Personal Website workshop.

When the workshop arc is ready, hopefully we can have a conversation about the workshops we'll work on next. Whether or not we use this, I think it helped me get the hang of writing these and understand the level of commitment needed. I feel bad for you having to make a workshop every week!

### CODE REVIEW

**Comment by zachlatta** [comment]
:-) will give this a read!

**Comment by jkwok91** [comment]
Hey @ad510! I just read the workshop and really enjoyed it.
I liked how you left some steps open so the reader could find solutions on their own or ask club members/leaders, as well as how you walked through and provided some sample solutions. I looked at your Bigfoot game before reading the tutorial, and I do think you should add in the part where Bigfoot moves around! Overall, this workshop was really clear, well organized, amusing to read, and therefore, awesome.

Other comments:
1) should the image src in the workshop be the url /img/bigfoot_image.png instead of the bigfoot.png on your website?
2) "It looks like whenever you put `onclick="myFunction()"` on a tag, it executes the [**JavaScript**](http://www.w3schools.com/js/) code you put in the quotes."
-- maybe rephrase to emphasize the relationship between clicking and "onclick":
"It looks like whenever you put `onclick="myFunction()"` on a tag, clicking on that element causes the JS code in quotes to be executed."

**Comment by zachlatta** [comment]
Assigning @jkwok91 because @jonleung doesn't seem to be actively working on this. Sorry for the accidental assign @jiahaok!

**Comment by sethtrei** [comment]
I like it so far except for the cloud nine thing. I don't love the username.github.io because maybe they dont want to make it ghpages. Also if they already did the gh-pages workshop they might get confused.

**Comment by zachlatta** [comment]
The current workshop progression for all new clubs starts with Personal
Website, which gets everyone set up with their main repository for Hack
Club as `username.github.io` (on GitHub Pages). Given that, I think we
should assume that all work will be done in `username.github.io` in future
workshops.

I know it's not super ideal and doesn't cater to every use case (especially
if they're concerned about privacy and don't want their stuff on GitHub
Pages), but I can't think of a better default than the current.

On Tue, Feb 9, 2016 at 4:11 PM, Seth Treiman <notifications@github.com>
wrote:

> I like it so far except for the cloud nine thing. I don't love the
> username.github.io because maybe they dont want to make it ghpages. Also
> if they already did the gh-pages workshop they might get confused.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/669#issuecomment-182141715>.
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>

**Comment by ad510** [comment]
I updated the workshop, and it should pass markcop after merging https://github.com/hackclub/markcop/pull/73.

I updated the image links assuming that https://github.com/ad510/workshop-assets will be copied to the hackclub org. See https://github.com/ad510/hackclub/pull/1#issuecomment-183548256 for my reasoning.

Anything else I have to do before merging?

**Comment by ad510** [comment]
@zachlatta @jkwok91 Any updates on this, and are you ok with how I did the image linking (see https://github.com/ad510/hackclub/pull/1#issuecomment-183548256 )?

Also, I realized that with some modifications, this can be turned into a tutorial on how to make a point and click adventure game. But let's get this merged first.

**Comment by zachlatta** [comment]
Sorry for the delay!

> I updated the image links assuming that
> https://github.com/ad510/workshop-assets will be copied to the hackclub
> org. See ad510#1 (comment) for my reasoning.
>
I really think relative links are the better way to go (especially
considering Markcop checks relative links and will know if they're broken).
Having a separate repository for assets will make the contribution process
more arduous for first-time contributors and and we don't get the benefits
of automated checks from a tool like Markcop.

> Also, I realized that with some modifications, this can be turned into a
> tutorial on how to make a point and click adventure game. But let's get
> this merged first.
>
Ah! This is a great idea. Biasing for merging this sounds good, but I'm
excited about having another workshop along those lines.

On Sat, Feb 13, 2016 at 10:57 AM, Andrew Downing <notifications@github.com>
wrote:

> @zachlatta <https://github.com/zachlatta> @jkwok91
> <https://github.com/jkwok91> Any updates on this, and are you ok with how
> I did the image linking (see ad510#1 (comment)
> <https://github.com/ad510/hackclub/pull/1#issuecomment-183548256> )?
>
> Also, I realized that with some modifications, this can be turned into a
> tutorial on how to make a point and click adventure game. But let's get
> this merged first.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/669#issuecomment-183723972>.
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>
